code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -56,6 +57,36 @@ Difficulty: Medium
 	. = ..()
 	transform *= 2
 
+/mob/living/simple_animal/hostile/megafauna/legion/enrage()
+	health = 1250
+	maxHealth = 1250
+	transform /= 1.5
+	loot = list(/datum/nothing)
+	crusher_loot = list(/datum/nothing)
+	var/mob/living/simple_animal/hostile/megafauna/legion/legiontwo = new /mob/living/simple_animal/hostile/megafauna/legion(get_turf(src))
+	legiontwo.transform /= 1.5
+	legiontwo.loot = list(/datum/nothing)
+	legiontwo.crusher_loot = list(/datum/nothing)
```

Comment:
Is there a specific reason these cant just be empty lists?
```suggestion
	legiontwo.loot = list()
	legiontwo.crusher_loot = list()
```

Reply:
megafauna runtime on death with no loot, iirc, #18768